server.js

```diff
@@ -9,6 +14,14 @@ const DB = process.env.DATABASE.replace(
 );
 mongoose.connect(DB).then(() => console.log("DB Connected successfully"));
 
-app.listen(process.env.PORT, () => {
+const server = app.listen(process.env.PORT, () => {
   console.log(`App running on port ${process.env.PORT}...`);
 });
+
+process.on("unhandledRejection", (err) => {
```

Comment:
Why do you want to shutdown the service for this? I am not sure I follow

Reply:
i followed a course for this and just did what he did, I will remove it.